rpcs3/rpcs3qt/save_manager_dialog.cpp

```diff
@@ -160,35 +219,49 @@ void save_manager_dialog::UpdateList()
 	int row = 0;
 	for (const SaveDataEntry& entry : m_save_entries)
 	{
+		QPixmap icon = QPixmap(320, 176);
+		if (!icon.loadFromData(entry.iconBuf.data(), static_cast<uint>(entry.iconBuf.size())))
+		{
+			LOG_WARNING(GENERAL, "Loading icon for save %s failed", entry.dirName);
+			icon.fill(QColor(0, 0, 0, 0));
+		}
+
 		QString title = qstr(entry.title);
-		QString subtitle = qstr(entry.subtitle);
+		title.append("\n");
+		title.append(qstr(entry.subtitle));
 		QString dirName = qstr(entry.dirName);
 
+		custom_table_widget_item* iconItem = new custom_table_widget_item;
+		iconItem->setData(Qt::UserRole, icon);
+		iconItem->setFlags(iconItem->flags() & ~Qt::ItemIsEditable);
+		m_list->setItem(row, 0, iconItem);
+
 		QTableWidgetItem* titleItem = new QTableWidgetItem(title);
 		titleItem->setData(Qt::UserRole, row); // For sorting to work properly
 		titleItem->setFlags(titleItem->flags() & ~Qt::ItemIsEditable);
+		m_list->setItem(row, 1, titleItem);
 
-		m_list->setItem(row, 0, titleItem);
-		QTableWidgetItem* subtitleItem = new QTableWidgetItem(subtitle);
-		subtitleItem->setFlags(subtitleItem->flags() & ~Qt::ItemIsEditable);
-		m_list->setItem(row, 1, subtitleItem);
+		QTableWidgetItem* modifiedItem = new QTableWidgetItem(FormatTimestamp(entry.mtime));
```

Comment:
Just my personal preference, but wouldn't it be better to call this timeItem or something like that?